libindy-crypto/src/cl/prover.rs

```diff
@@ -322,19 +322,19 @@ impl ProofBuilder {
         values.extend_from_slice(&self.c_list);
         values.push(nonce.to_bytes()?);
 
-        let c_h = get_hash_as_int(&mut values)?;
+        let challenge = get_hash_as_int(&mut values)?;
```

Comment:
Please put a comment that `challenge ` is denoted by `c_h` in the paper

Reply:
Done